systers_portal/templates/meetup/upcoming_meetups.html

```diff
@@ -0,0 +1,15 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+  <div class="mt20 mb40">
+    <h2 class="ml15"> Upcoming Meetups </h2>
+      {% for meetup in meetup_list %}
+        <div class="mt20 ml15">
+          <h4>
```

Comment:
It would be nice to display some extra info besides the title. The title only doesn't give much insight into what's the meetup about. Same holds for past events :)
Inspiration: http://www.meetup.com/PyLadies-Berlin/ (look at the Upcoming tab or below that Recents).